bin/bindings.properties

```diff
@@ -58,7 +58,8 @@ jdbc:site.ycsb.db.JdbcDBClient
 kudu:site.ycsb.db.KuduYCSBClient
 memcached:site.ycsb.db.MemcachedClient
 mongodb:site.ycsb.db.MongoDbClient
-mongodb-async:site.ycsb.db.AsyncMongoDbClient
+mongodb2:site.ycsb.db.MongoDb2Client
```

Comment:
@RaviTella  is this MongoDB2Client is needed here and in bin/ycsb file ?

Reply:
it's not required. Removed it from bindings.properties and replaced it with "mongodbreactivestreams" binding ref in ycsb python file.